model/src/test/kotlin/config/RepositoryConfigurationTest.kt

```diff
@@ -112,6 +112,11 @@ class RepositoryConfigurationTest : WordSpec() {
                 scopes.first().name.pattern shouldBe "scope"
                 scopes.first().reason shouldBe ScopeExcludeReason.TEST_TOOL_OF
                 scopes.first().comment shouldBe "scope comment"
+
+                val error = repositoryConfiguration.resolutions!!.errors.first()
```

Comment:
I don't wanna add those because adding the above two lines will not provide you any additional information in case the test fails.

Reply:
Are you ok with that? I had a similar discussion on Sebastians PR yesterday with the outcome of not adding such assertions.

Reply:
For the null check that might be valid, even if I would still prefer the test framework to handle this and throw an `AssertionError` instead of having a `NullPointerException` (e.g. makes the test yellow instead of red if running in IntelliJ). I think you should not test every value for `null` if this is not part of the scenario you are testing, but in this case the line means to deliberately test if `resolutions` is correctly deserialized and not ignored. But in the end this is of course also tested without this line, just not as explicitly.

However, the second line does add value, because it makes sure that the list does not contain more than one entry, which would be an error we would miss otherwise.

Reply:
yes, I added the size check to be on the save side, however the null given the reasons mentioned I still see no value in adding this. 
Consider that this yellow case won't ever make it into master